plugin.video.funkmediathek/addon.xml

```diff
@@ -12,6 +13,7 @@
 		<language>de</language>
 		<license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
 		<source>https://github.com/prof-membrane/plugin.video.funkmediathek</source>
+		<forum>http://forum.kodi.tv/showthread.php?tid=303100</forum>
```

Comment:
Thanks for the update, I noticed you are missing a assets parameter. If you could add this to your next update I'd appreciate it. THX
https://kodi.wiki/view/Addon.xml#.3Cassets.3E